solutions/project/project-sandbox/network/route.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# A Route to the internet that requires that the resources attached to the network
+# specify it's tag to access the internet
+# SC-7(5) BOUNDARY PROTECTION | DENY BY DEFAULT / ALLOW BY EXCEPTION
+apiVersion: compute.cnrm.cloud.google.com/v1beta1
+kind: ComputeRoute
+metadata:
+  name: project-id-internet-route # kpt-set: ${project-id}-internet-route
```

Comment:
Just a small suggestion to change this to `project-id-internet-egress-route`.

Reply:
can you please specify what value you would also like to see for the resourceID and the tag. below is what we had in the terraform landing zone
routes:
  - route_name: "egress-internet-p3nonp"
    description: "route through IGW to access internet"
    destination_range: "0.0.0.0/0"
    tags: "egress-inet"
    next_hop_internet: true

Reply:
I was more concerned about having a destination description in the name such as egress/ingress. The example above `egress-internet-p3nonp` is good since it contains a destination - `egress`. Let me know if that makes sense.

Reply:
My comment comes from working with the SSC naming convention. The idea is to have routes contain source and destination - ex: <source device>-to-<destination>-route. Since we're only using the naming convention for projects at the moment, these are good `egress-internet-p3nonp` / `egress-inet`